sonaranalyzer-dotnet/src/SonarAnalyzer.VisualBasic/Metrics/VisualBasicExecutableLinesMetric.cs

```diff
@@ -88,6 +88,7 @@ private bool FindExecutableLines(SyntaxNode node)
 
                     case SyntaxKind.EmptyStatement:
                     case SyntaxKind.ExpressionStatement:
+                    case SyntaxKind.SimpleAssignmentStatement:
```

Comment:
In VB.NET, `SimpleAssignmentStatement` is not wrapped within an `ExpressionStatement` like it is for C#.